app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -101,15 +101,30 @@ export const ContentListView: React.FC = () => {
     navigate(`/contents/combined/${content.id}`);
   };
 
+  const hideColumns = (combined: boolean, contentType?: ContentTypeName) => {
```

Comment:
Cleaner way to hide columns

Reply:
I had something like this last night, but made it much more confusing due to fatigue and undid my changes 😝